pilot/pkg/proxy/envoy/testdata/external-service-rule-tcp-dns.yaml.golden

```diff
@@ -0,0 +1,11 @@
+hosts:
+  - "172.217.0.0/16"
+ports:
+- number: 444
+  name: tcp-444
+  protocol: tcp
+discovery: DNS
+endpoints: # TODO endpoints not required according to spec, but they would be here because the host field is not a DNS resolvable address
```

Comment:
actually, this is fine.. I wouldn't mark this as TODO though.

Reply:
right. it should be handled by validation now.